app/controllers/api_controller/queue.rb

```diff
@@ -0,0 +1,21 @@
+class ApiController
+  INVALID_QUEUE_ATTRS = %w(id href state lock_version)
```

Comment:
This feels backwards...we should whitelist, not blacklist, because adding a new field to the queue table would auto-expose it here.